dataeng/resources/token-helper.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
```

Comment:
This file needs to be executable. `chmod +x token-helper.py` should take care of that.

Reply:
Earlier we didn't made this executable still it was getting executed referring to this one http://jenkins.analytics.edx.org/job/analytics-exporter-worker/17946/console I am wondering do we need this in this case